### PR TITLE
Add etcd_debug and etcd_log_package_levels variables

### PR DESCRIPTION
**Pull request by sdodson**
Help debug https://github.com/openshift/origin/issues/13067

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

File: roles/etcd/templates/etcd.conf.j2 [hunk 1]
```diff
@@ -60,3 +60,10 @@ ETCD_PEER_CA_FILE={{ etcd_peer_ca_file }}
 ETCD_PEER_CERT_FILE={{ etcd_peer_cert_file }}
 ETCD_PEER_KEY_FILE={{ etcd_peer_key_file }}
 {% endif -%}
+
+#[logging]
+ETCD_DEBUG="{{ 'true' if etcd_debug | default(false) else 'false' }}"
```
Comment (stevekuznetsov) [inline roles/etcd/templates/etcd.conf.j2]: Just cast to a string?

```
ETCD_DEBUG="{{ etcd_debug | default(false) | string }}"
```

File: roles/etcd/templates/etcd.conf.j2 [hunk 2]
```diff
@@ -60,3 +60,10 @@ ETCD_PEER_CA_FILE={{ etcd_peer_ca_file }}
...
+# examples for -log-package-levels etcdserver=WARNING,security=DEBUG
+{% if etcd_log_package_levels is defiend %}
```
Comment (stevekuznetsov) [inline roles/etcd/templates/etcd.conf.j2]: `s/defiend/defined`

File: roles/etcd/templates/etcd.conf.j2 (hunk 1 above)
Comment (sdodson) [inline roles/etcd/templates/etcd.conf.j2]: Ok, we've done it the original way elsewhere but it's not exactly a pattern.

Comment (sdodson) [inline roles/etcd/templates/etcd.conf.j2]: Hmm, etcd_debug=0 yields 'false' or '0'?

**Comment by ingvagabund** [comment]
Complying to [1].

[1] https://github.com/coreos/etcd/blob/20490caaf0dcd96bb4a95e40625559def8ef5b04/Documentation/op-guide/configuration.md#logging-flags

File: roles/etcd/templates/etcd.conf.j2 [hunk 3]
```diff
@@ -60,3 +60,10 @@ ETCD_PEER_CA_FILE={{ etcd_peer_ca_file }}
 ETCD_PEER_CERT_FILE={{ etcd_peer_cert_file }}
 ETCD_PEER_KEY_FILE={{ etcd_peer_key_file }}
 {% endif -%}
+
+#[logging]
+ETCD_DEBUG="{{ etcd_debug | default(false) | string }}"
+# examples for -log-package-levels etcdserver=WARNING,security=DEBUG
```
Comment (stevekuznetsov) [inline roles/etcd/templates/etcd.conf.j2]: Maybe remove this, just clutter -- someone should be in the `etcd` doc if they're touching this.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Pull request merged**